joomla/joomla-cms
# [4.0] Namespace com_menus

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_menus component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_menus.zip )

2. Access to backend, try to add/edit/publish/unpublish... menu and menu items, make sure it is all working as expected (works in the same way as before it is namespaced)
